src/styles/blocks/catalog.scss

```diff
@@ -21,5 +21,12 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition: $trn-duration;
+    &:hover {
+      color: $c-cardOnHover;
+      transform: scale(1.2);
+      cursor: pointer;
```

Comment:
Keep this prop with element, not pseudo-class

Reply:
fixed